client-common/src/main/java/org/apache/cassandra/sidecar/common/response/ListCdcSegmentsResponse.java

```diff
@@ -65,4 +66,19 @@ public List<CdcSegmentInfo> segmentInfos()
     {
         return segmentInfos;
     }
+
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o) return true;
```

Comment:
Does this conform to the project code style? I think braces are required.

Reply:
There is a check style check that is run on every build, since the CircleCi is green, I don't think that this is an issue.

Reply:
Unfortunately, requiring braces for single line if-else body is not declared in the check style ruleset. I believe most of the if-else statements do have braces for the single line body. Please stay consistent. We can add the check style rule to enforce it.